examples/all-clusters-app/linux/BUILD.gn

```diff
@@ -15,6 +15,28 @@
 import("//build_overrides/build.gni")
 import("//build_overrides/chip.gni")
 
+static_library("static-supported-modes-manager") {
```

Comment:
Why this separate library instad of just adding this to the sources for `executable("chip-all-clusters-app")`?